Frontend/src/components/Search/SearchContent/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+import { useRouteMatch, Switch, Route } from "react-router-dom";
+import SearchResultList from "components/Search/SearchResultList";
+import TeamCard from "components/Search/SearchResultList/TeamCard";
+import InstitutionCard from "components/Search/SearchResultList/InstitutionCard";
+import UserCard from "components/Search/SearchResultList/UserCard";
+import ProjectCard from "components/Search/SearchResultList/ProjectCard";
+import ResultStatus from "components/Search/ResultStatus";
+import { setTypeInUrl } from "shared/utils/common/utils";
+
+
+// wrapper component to display the searched items  
+const SearchContent = ({ searchedData, type, searchedKeyword }) => {
+  const { path } = useRouteMatch();
+
+  const renderSearchResultsList = (listItemWrapper) => {
+    return (
+      <SearchResultList
+        listItemWrapper={listItemWrapper}
+        searchedData={searchedData}
+      />
+    );
+  };
+
+  return (
+    <React.Fragment>
+      <ResultStatus
+        count={searchedData.length}
+        searchedEntity={type}
+        searchedKeyword={searchedKeyword}
+      />
+      <Switch>
+        <Route path={setTypeInUrl(path, "teams")}>
```

Comment:
Why do not you build up the string directly? but you use a function for it. You can leave it as it is but just wanted to raise it.

Reply:
So my goal was avoid hardcoding <Route path="..."/> so if the route managed in DashbordContent is changed, the component is not/less affected. Maybe there is a cleaner/in-build solution for that, but I couldn't find it.